CreateService/handler.ts

```diff
@@ -123,20 +124,55 @@ const createSubscriptionTask = (
             : taskEither.of(responseType.value)
       )
   );
+
+const getUserTask = (
+  logger: ILogger,
+  apiClient: ReturnType<APIClient>,
+  userEmail: EmailString
+): TaskEither<ErrorResponses, UserInfo> =>
+  tryCatch(
+    () =>
+      apiClient.getUser({
+        email: userEmail
+      }),
+    errs => {
+      logger.logUnknown(errs);
+      return toDefaultResponseErrorInternal(errs);
+    }
+  ).foldTaskEither(
+    err => fromLeft(err),
+    errorOrResponse =>
+      errorOrResponse.fold(
+        errs => {
+          logger.logErrors(errs);
+          return fromLeft(toDefaultResponseErrorInternal(errs));
+        },
+        responseType =>
+          responseType.status !== 200
+            ? fromLeft(toErrorServerResponse(responseType))
+            : taskEither.of(responseType.value)
+      )
+  );
```

Comment:
Since this looks like a common pattern, may you try, with another PR, to write a generic wrapper that takes as input the API method ie.

```ts
const getUserTask = (logger: ILogger, 
  apiClient: ReturnType<APIClient>, 
  userEmail: EmailString) => 
     withApiRequestWrapper(logger, () => apiClient.getUser({
        email: userEmail
      }), 200);
```

where "200" is the success http status code (may be 200 | 201 | 202)